pages/_app.tsx

```diff
@@ -2,15 +2,36 @@ import { ChakraProvider } from '@chakra-ui/react';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 import type { AppProps } from 'next/app';
+import Head from 'next/head';
+import Script from 'next/script';
+import { useEffect } from 'react';
 
 import { heyTheme } from '@/public/theme/theme';
 
 const queryClient = new QueryClient();
 
+declare global {
+  interface Window {
+    Kakao: any;
+  }
+}
+
 export default function App({ Component, pageProps }: AppProps) {
+  function kakaoInit() {
+    window.Kakao.init(process.env.NEXT_PUBLIC_KAKAO_KEY);
+  }
+
   return (
     <QueryClientProvider client={queryClient}>
       <ChakraProvider theme={heyTheme}>
+        <Head>
+          <title>Hey, cake</title>
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+        </Head>
+        <Script
+          src="https://t1.kakaocdn.net/kakao_js_sdk/2.1.0/kakao.min.js"
+          onLoad={kakaoInit}
+        ></Script>
```

Comment:
우리 단축하는건 어떨까요? 

```suggestion
        />
```

이렇게요 ㅎㅎ